apps/taxes/src/pages/providers/avatax/[id].tsx

```diff
@@ -12,7 +13,9 @@ const EditAvataxPage = () => {
     <main>
       <AppColumns top={<Header />}>
         <AvataxInstructions />
-        <EditAvataxConfiguration />
+        <Provider>
```

Comment:
why this is not in `_app` ?

Reply:
It's not a general, per-app provider. The Providers in jotai [clear the state of atoms](https://jotai.org/docs/core/provider). We want to have a separate state of atoms per each edit/create page.